### DISCUSSION TITLE
[Community localization checklist] Add warning when a component does not have any user receiving notifications for repository errors

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#10317] [Ideas] [answered]
In hosted.weblate.org and maybe in other weblate instances too, some components are not actively mantained, or have lost connection to their upstreams and are failing.

For translators this is quite frustrating, see for example https://github.com/WeblateOrg/weblate/discussions/8276 but there are many instances.

Oftentimes this is not the intention of the maintainers, but there is a lack of communication or they haven't realised they need to set up notifications for the component.

So, I suggest that the Community Localization Checklist provide a warning on the "Workflow customization" section when there are no notifications setup on a component for:

- Component was locked or unlocked
- Parse error occured
- New language was added or requested
- New alert emerged in a component
- Repository operation failed

This can also be accompanied by a list to the user's profile notification settings offering 'set this notifications' or something like that.

**Comment by nijel** [comment, marked as answer]
I think this makes sense, created https://github.com/WeblateOrg/weblate/issues/10654 as an issue for this.
